flow-ide.py

```diff
@@ -127,7 +163,32 @@ def call_flow_cli(contents, command):
         except:
             os.close(read)
             print(e.output)
+            view.erase_regions('flow_error')
+            view.erase_regions('flow_uncovered')
+            view.set_status('flow_error', 'Unknown Flow error: ' + str(e.output))
             return None
+    except Exception as e:
+        os.close(read)
+        view.erase_regions('flow_error')
```

Comment:
This looks copied from above. Can we move this to a utility function like `reset()` or something?